.eslintignore

```diff
@@ -6,5 +6,7 @@ node_modules/
 
 # TODO: don't ignore these examples once they're updated
 /examples/calendar-widget
+/examples/e-commerce/.cache
```

Comment:
More globally, we're safe to remove the `/examples/e-commerce/` prefix for these two lines so that we're not surprised again when we migrate the other examples. We can change those two lines above:

```diff
- /dist
+ dist
+ .cache
```

Reply:
@francoischalifour Thanks for the feedback. Just updated the pull request.